src/components/calculator.js

```diff
@@ -1,16 +1,9 @@
 import React from 'react';
 import CalculatorLayout from './layout';
 
-export default class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+export default function Calculator() {
+  return (
+    <CalculatorLayout />
```

Comment:
- [ ] It would be better if you use the arrow function instead of the keyword function. kindly fix this minor issue. 

```suggestion
const Calculator = () => (
  return (
    <CalculatorLayout />
```

Reply:
Thank you for making that statement. I have refactored it with arrow functions